packages/suite-desktop-core/e2e/support/pageActions/recoverActions.ts

```diff
@@ -0,0 +1,13 @@
+import { Locator, Page } from '@playwright/test';
+
+export class RecoverActions {
+    readonly selectBasicRecoveryButton: Locator;
+
+    constructor(private page: Page) {
+        this.selectBasicRecoveryButton = page.getByTestId('@recover/select-type/basic');
+    }
+
+    async selectWordCount(number: number) {
```

Comment:
There are 3 options right? Lets user literal union as type to make the use of this method easier.

``` suggestion
    async selectWordCount(number: 12 | 16| 24 ) {
```